portfolio/src/main/webapp/index.html

```diff
@@ -36,21 +36,16 @@ <h2> "A Little Introduction"</h2>
             It cemented my interests in computer science and the memeories I made over that summer were
             unforgettable.
         </p>
+        <p>
            Althoguh I'm only a Rising Junior nor as well as the fcat that I'm nowhere near the smartest person
            I know, I want this page to represent my progress as I grow. Becuase I want to share with the world
           all the places that I will go.
         </p>
     </div>
 
       <p>Click here to get a brief overview of who I am :</p>
-      <button onclick="addRandomFact()">One fact about me is </button>
+      <button onclick="addRandomFact(); introduce(); fetchData()">One fact about me is </button>
```

Comment:
I am not sure the best practice of JS, but I think it might be better if we call just one "main" function, where it calls these sub-functions. I think it also aligns with the concept of separating html and script

Reply:
I was thinking of just chaning the html to caall the fetch data fucntion onload

Reply:
+1 to Tiancong's suggestion. I wouldn't include more than one method call in an onclick attribute.